calendar-bundle/src/Generator/CalendarEventsGenerator.php

```diff
@@ -103,9 +104,9 @@ public function getAllEvents(array $calendars, \DateTimeInterface $rangeStart, \
         }
 
         // HOOK: modify the result set
-        if (isset($GLOBALS['TL_HOOKS']['getAllEvents']) && \is_array($GLOBALS['TL_HOOKS']['getAllEvents'])) {
+        if ($module && isset($GLOBALS['TL_HOOKS']['getAllEvents']) && \is_array($GLOBALS['TL_HOOKS']['getAllEvents'])) {
```

Comment:
```suggestion
        if (isset($GLOBALS['TL_HOOKS']['getAllEvents']) && \is_array($GLOBALS['TL_HOOKS']['getAllEvents'])) {
```

Shouldn't the hook also be executed if `$module` is `null`?

Reply:
No it cannot be executed if there is no module. The hook expects the module.